ios/RCTWebRTC/VideoCaptureController.h

```diff
@@ -1,17 +1,23 @@
 
 #import <Foundation/Foundation.h>
-#import <WebRTC/RTCCameraVideoCapturer.h>
+
+#if !TARGET_OS_TV
+    #import <WebRTC/RTCCameraVideoCapturer.h>
+#endif
 
 #import "CaptureController.h"
 
 @interface VideoCaptureController : CaptureController
-
-@property(nonatomic, readonly, strong) AVCaptureDeviceFormat *selectedFormat;
+#if !TARGET_OS_TV
+    @property(nonatomic, readonly, strong) AVCaptureDeviceFormat *selectedFormat;
```

Comment:
Note for future reference: https://developer.apple.com/documentation/avfoundation/avcapturedeviceformat is supported on tvOS 17.